src/server/page.tsx

```diff
@@ -107,6 +108,7 @@ export const getPageMiddleware: Koa.Middleware = async (ctx) => {
 
   if (!story) {
     ctx.status = 404
+    appLogger.info(`Request returned 404 due to missing storyblok story`)
```

Comment:
NICE with logging here! However this should prolly be the request logger rather than the `appLogger`. The difference is basically that it prefixes with a request id, so you can track all log posts from a specific request through all the systems (i.e. it intercepts the `X-Request-Id` header from heroku).  example: https://github.com/HedvigInsurance/web-next/blob/8bb147b119f845128d68962b7d672d3236d68d02/src/server/page.tsx#L77